2019-12-04 05:14:08 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 05:14:08 - root - INFO: - Feature Engineering at 2019-12-03 has not done. Run the whole process.
2019-12-04 05:14:08 - root - INFO: - Removing previous log and results 30 ago
2019-12-04 05:14:08 - root - WARNING: - /inv_check_daily_prediction/output/log/log_20191103 Not Exists or removed
2019-12-04 05:14:08 - root - WARNING: - /inv_check_daily_prediction/output/prediction/prediction_20191103.csv Not Exists or removed
2019-12-04 05:14:08 - root - WARNING: - /inv_check_daily_prediction/output/feature/feature_20191103.csv Not Exists or removed
2019-12-04 05:14:08 - root - INFO: - Query Data From ODS.Opendata at 2019-12-03
2019-12-04 05:14:28 - root - INFO: - Filling Missing Time at 2019-12-03
2019-12-04 05:14:32 - root - INFO: - Merging Stock Data with Index Data at 2019-12-03
2019-12-04 05:14:36 - root - INFO: - Reading column dict at 2019-12-03
2019-12-04 05:14:36 - root - INFO: - Feature Engineering at 2019-12-03
2019-12-04 05:17:39 - root - INFO: - Reading feature list at 2019-12-03
2019-12-04 05:17:40 - root - INFO: - Predicting at 2019-12-03
2019-12-04 05:18:26 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 05:18:26 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 05:18:32 - root - INFO: - Done at 2019-12-03
2019-12-04 05:18:32 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 05:20:23 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 05:20:24 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 05:20:24 - root - INFO: - Predicting at 2019-12-03
2019-12-04 05:21:10 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 05:21:10 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 05:21:15 - root - INFO: - Done at 2019-12-03
2019-12-04 05:21:15 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 05:28:10 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 05:28:10 - root - INFO: - Feature Engineering at 2019-12-02 has not done. Run the whole process.
2019-12-04 05:28:10 - root - INFO: - Removing previous log and results 30 ago
2019-12-04 05:28:10 - root - WARNING: - /inv_check_daily_prediction/output/log/log_20191102 Not Exists or removed
2019-12-04 05:28:10 - root - WARNING: - /inv_check_daily_prediction/output/prediction/prediction_20191102.csv Not Exists or removed
2019-12-04 05:28:10 - root - WARNING: - /inv_check_daily_prediction/output/feature/feature_20191102.csv Not Exists or removed
2019-12-04 05:28:10 - root - INFO: - Query Data From ODS.Opendata at 2019-12-02
2019-12-04 05:52:32 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 05:52:32 - root - INFO: - Feature Engineering at 2019-12-02 has not done. Run the whole process.
2019-12-04 05:52:32 - root - INFO: - Removing previous log and results 30 ago
2019-12-04 05:52:32 - root - WARNING: - /inv_check_daily_prediction/output/log/log_20191102 Not Exists or removed
2019-12-04 05:52:32 - root - WARNING: - /inv_check_daily_prediction/output/prediction/prediction_20191102.csv Not Exists or removed
2019-12-04 05:52:32 - root - WARNING: - /inv_check_daily_prediction/output/feature/feature_20191102.csv Not Exists or removed
2019-12-04 05:52:32 - root - INFO: - Query Data From ODS.Opendata at 2019-12-02
2019-12-04 05:52:51 - root - INFO: - Filling Missing Time at 2019-12-02
2019-12-04 05:52:55 - root - INFO: - Merging Stock Data with Index Data at 2019-12-02
2019-12-04 05:52:59 - root - INFO: - Reading column dict at 2019-12-02
2019-12-04 05:52:59 - root - INFO: - Feature Engineering at 2019-12-02
2019-12-04 05:56:07 - root - INFO: - Reading feature list at 2019-12-02
2019-12-04 05:56:08 - root - INFO: - Predicting at 2019-12-02
2019-12-04 05:56:54 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 05:56:54 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 05:56:54 - root - ERROR: - Exception: (208, b"Invalid object name 'PREDICTION'.DB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n")
2019-12-04 05:56:54 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "src/pymssql.pyx", line 448, in pymssql.Cursor.execute
  File "src/_mssql.pyx", line 1064, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1095, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1228, in _mssql.MSSQLConnection.format_and_run_query
  File "src/_mssql.pyx", line 1639, in _mssql.check_cancel_and_raise
  File "src/_mssql.pyx", line 1683, in _mssql.maybe_raise_MSSQLDatabaseException
_mssql.MSSQLDatabaseException: (208, b"Invalid object name 'PREDICTION'.DB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 296, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 56, in write_to_db
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 46, in send_query
    odscur.execute(query)
  File "src/pymssql.pyx", line 465, in pymssql.Cursor.execute
pymssql.ProgrammingError: (208, b"Invalid object name 'PREDICTION'.DB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n")

2019-12-04 05:58:32 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 05:58:32 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 05:58:32 - root - INFO: - Predicting at 2019-12-02
2019-12-04 05:59:19 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 05:59:19 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 05:59:19 - root - ERROR: - Exception: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')
2019-12-04 05:59:19 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "src/pymssql.pyx", line 448, in pymssql.Cursor.execute
  File "src/_mssql.pyx", line 1064, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1095, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1228, in _mssql.MSSQLConnection.format_and_run_query
  File "src/_mssql.pyx", line 1639, in _mssql.check_cancel_and_raise
  File "src/_mssql.pyx", line 1683, in _mssql.maybe_raise_MSSQLDatabaseException
_mssql.MSSQLDatabaseException: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 55, in write_to_db
    _, _ = send_query(remove_query)
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 46, in send_query
    odscur.execute(query)
  File "src/pymssql.pyx", line 468, in pymssql.Cursor.execute
pymssql.OperationalError: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

2019-12-04 06:00:57 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:00:57 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:00:57 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:01:44 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:01:44 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:01:44 - root - ERROR: - Exception: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')
2019-12-04 06:01:44 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "src/pymssql.pyx", line 448, in pymssql.Cursor.execute
  File "src/_mssql.pyx", line 1064, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1095, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1228, in _mssql.MSSQLConnection.format_and_run_query
  File "src/_mssql.pyx", line 1639, in _mssql.check_cancel_and_raise
  File "src/_mssql.pyx", line 1683, in _mssql.maybe_raise_MSSQLDatabaseException
_mssql.MSSQLDatabaseException: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 55, in write_to_db
    _, _ = send_query(remove_query)
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 46, in send_query
    odscur.execute(query)
  File "src/pymssql.pyx", line 468, in pymssql.Cursor.execute
pymssql.OperationalError: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

2019-12-04 06:02:19 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:02:20 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:02:20 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:03:06 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:03:06 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:03:06 - root - ERROR: - Exception: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')
2019-12-04 06:03:06 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "src/pymssql.pyx", line 448, in pymssql.Cursor.execute
  File "src/_mssql.pyx", line 1064, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1095, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1228, in _mssql.MSSQLConnection.format_and_run_query
  File "src/_mssql.pyx", line 1639, in _mssql.check_cancel_and_raise
  File "src/_mssql.pyx", line 1683, in _mssql.maybe_raise_MSSQLDatabaseException
_mssql.MSSQLDatabaseException: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 55, in write_to_db
    _, _ = send_query(remove_query)
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 46, in send_query
    odscur.execute(query)
  File "src/pymssql.pyx", line 468, in pymssql.Cursor.execute
pymssql.OperationalError: (206, b'Operand type clash: date is incompatible with intDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n')

2019-12-04 06:04:37 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:04:37 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:04:37 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:05:24 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:05:24 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:05:24 - root - ERROR: - Exception: Statement not executed or executed statement has no resultset
2019-12-04 06:05:24 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 55, in write_to_db
    _, _ = send_query(remove_query)
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 47, in send_query
    temp = odscur.fetchall()
  File "src/pymssql.pyx", line 543, in pymssql.Cursor.fetchall
pymssql.OperationalError: Statement not executed or executed statement has no resultset

2019-12-04 06:08:47 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:08:47 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:08:47 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:09:34 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:09:34 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:09:34 - root - ERROR: - Exception: Statement not executed or executed statement has no resultset
2019-12-04 06:09:34 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 55, in write_to_db
    _, _ = send_query(remove_query)
  File "/inv_check_daily_prediction/core/ALL_STOCK_preprocess_function.py", line 49, in send_query
    temp = odscur.fetchall()
  File "src/pymssql.pyx", line 543, in pymssql.Cursor.fetchall
pymssql.OperationalError: Statement not executed or executed statement has no resultset

2019-12-04 06:10:17 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:10:17 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:10:17 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:11:04 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:11:04 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:11:10 - root - INFO: - Done at 2019-12-02
2019-12-04 06:11:10 - root - INFO: - Sending message and result to telegram at 2019-12-02
2019-12-04 06:17:20 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 06:17:20 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 06:17:20 - root - INFO: - Predicting at 2019-12-02
2019-12-04 06:18:06 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 06:18:06 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 06:18:12 - root - INFO: - Done at 2019-12-02
2019-12-04 06:18:12 - root - INFO: - Sending message and result to telegram at 2019-12-02
2019-12-04 06:19:10 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 06:19:10 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 06:19:10 - root - INFO: - Predicting at 2019-12-03
2019-12-04 06:19:56 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 06:19:56 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 06:20:02 - root - INFO: - Done at 2019-12-03
2019-12-04 06:20:02 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 06:22:07 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 06:22:08 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 06:22:08 - root - INFO: - Predicting at 2019-12-03
2019-12-04 06:22:54 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 06:22:55 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 06:22:55 - root - ERROR: - Exception: (102, b"Incorrect syntax near '?'.DB-Lib error message 20018, severity 15:\nGeneral SQL Server error: Check messages from the SQL Server\n")
2019-12-04 06:22:55 - root - ERROR: - Failed when Writing Prediction to Database 
Traceback (most recent call last):
  File "src/pymssql.pyx", line 450, in pymssql.Cursor.execute
  File "src/_mssql.pyx", line 1064, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1095, in _mssql.MSSQLConnection.execute_query
  File "src/_mssql.pyx", line 1228, in _mssql.MSSQLConnection.format_and_run_query
  File "src/_mssql.pyx", line 1639, in _mssql.check_cancel_and_raise
  File "src/_mssql.pyx", line 1683, in _mssql.maybe_raise_MSSQLDatabaseException
_mssql.MSSQLDatabaseException: (102, b"Incorrect syntax near '?'.DB-Lib error message 20018, severity 15:\nGeneral SQL Server error: Check messages from the SQL Server\n")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/inv_check_daily_prediction/main.py", line 106, in main
    Predict.write_to_db(result_df, config['writing_table'])
  File "/inv_check_daily_prediction/core/Prediction.py", line 70, in write_to_db
    odscur.execute(remove_query, ('2019-12-03',))
  File "src/pymssql.pyx", line 465, in pymssql.Cursor.execute
pymssql.ProgrammingError: (102, b"Incorrect syntax near '?'.DB-Lib error message 20018, severity 15:\nGeneral SQL Server error: Check messages from the SQL Server\n")

2019-12-04 06:23:37 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 06:23:37 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 06:23:37 - root - INFO: - Predicting at 2019-12-03
2019-12-04 06:24:23 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 06:24:23 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 06:24:29 - root - INFO: - Done at 2019-12-03
2019-12-04 06:24:29 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 06:25:48 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 06:25:48 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 06:25:48 - root - INFO: - Predicting at 2019-12-03
2019-12-04 06:26:34 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 06:26:34 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 06:26:40 - root - INFO: - Done at 2019-12-03
2019-12-04 06:26:40 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 06:28:01 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 06:28:01 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 06:28:01 - root - INFO: - Predicting at 2019-12-03
2019-12-04 06:28:49 - root - INFO: - Writing to Local at 2019-12-03
2019-12-04 06:28:49 - root - INFO: - Writing to Database at 2019-12-03
2019-12-04 06:28:54 - root - INFO: - Done at 2019-12-03
2019-12-04 06:28:54 - root - INFO: - Sending message and result to telegram at 2019-12-03
2019-12-04 07:13:31 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 07:13:31 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 07:13:31 - root - INFO: - Predicting at 2019-12-03
2019-12-04 15:22:32 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-03
2019-12-04 15:22:32 - root - INFO: - Feature Engineering of 2019-12-03 has done. Start to predict directly.
2019-12-04 15:22:32 - root - INFO: - Predicting at 2019-12-03
2019-12-04 18:11:11 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 18:11:11 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 18:11:11 - root - INFO: - Predicting at 2019-12-02
2019-12-04 18:11:32 - retry.api - WARNING: - HTTPConnectionPool(host='128.110.238.61', port=8080): Max retries exceeded with url: /predict (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3c301a3ba8>: Failed to establish a new connection: [Errno 111] Connection refused')), retrying in 120 seconds...
2019-12-04 18:12:48 - root - INFO: - Checking if Feature Engineering is already done at 2019-12-02
2019-12-04 18:12:48 - root - INFO: - Feature Engineering of 2019-12-02 has done. Start to predict directly.
2019-12-04 18:12:48 - root - INFO: - Predicting at 2019-12-02
2019-12-04 18:14:01 - root - INFO: - Writing to Local at 2019-12-02
2019-12-04 18:14:01 - root - INFO: - Writing to Database at 2019-12-02
2019-12-04 18:14:07 - root - INFO: - Done at 2019-12-02
2019-12-04 18:14:07 - root - INFO: - Sending message and result to telegram at 2019-12-02
2019-12-04 18:15:52 - root - INFO: - Checking if Feature Engineering is already done at 2019-11-29
2019-12-04 18:15:52 - root - INFO: - Feature Engineering at 2019-11-29 has not done. Run the whole process.
2019-12-04 18:15:52 - root - INFO: - Removing previous log and results 30 ago
2019-12-04 18:15:52 - root - WARNING: - /inv_check_daily_prediction/output/log/log_20191030 Not Exists or removed
2019-12-04 18:15:52 - root - WARNING: - /inv_check_daily_prediction/output/prediction/prediction_20191030.csv Not Exists or removed
2019-12-04 18:15:52 - root - WARNING: - /inv_check_daily_prediction/output/feature/feature_20191030.csv Not Exists or removed
2019-12-04 18:15:52 - root - INFO: - Query Data From ODS.Opendata at 2019-11-29
2019-12-04 18:16:05 - retry.api - WARNING: - Stock data length doesn't match, retrying in 300 seconds...
2019-12-04 18:23:04 - root - INFO: - Checking if Feature Engineering is already done at 2019-11-29
2019-12-04 18:23:04 - root - INFO: - Feature Engineering at 2019-11-29 has not done. Run the whole process.
2019-12-04 18:23:04 - root - INFO: - Removing previous log and results 30 ago
2019-12-04 18:23:04 - root - WARNING: - /inv_check_daily_prediction/output/log/log_20191030 Not Exists or removed
2019-12-04 18:23:04 - root - WARNING: - /inv_check_daily_prediction/output/prediction/prediction_20191030.csv Not Exists or removed
2019-12-04 18:23:04 - root - WARNING: - /inv_check_daily_prediction/output/feature/feature_20191030.csv Not Exists or removed
2019-12-04 18:23:04 - root - INFO: - Query Data From ODS.Opendata at 2019-11-29
2019-12-04 18:23:22 - root - INFO: - Filling Missing Time at 2019-11-29
2019-12-04 18:23:30 - root - INFO: - Merging Stock Data with Index Data at 2019-11-29
2019-12-04 18:23:38 - root - INFO: - Reading column dict at 2019-11-29
2019-12-04 18:23:38 - root - INFO: - Feature Engineering at 2019-11-29
